pyproject.toml

```diff
@@ -0,0 +1,61 @@
+[build-system]
+requires = ["setuptools>=61.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "kafe2"
+description = "Karlsruhe Fit Environment 2: a package for fitting and elementary data analysis"
+authors = [
+	{ name="Daniel Savoiu", email="daniel.savoiu@cern.ch" },
```

Comment:
While you're at it...
You could add the following authors:
```
{name="Johannes Gäßler", email="johannesg@5d6.de"},
{name="Cedric Verstege", email="cedric.verstege@cern.ch"},
{name="Günter Quast, email="G.Quast@kit.edu"},
```
I hope @JohannesGaessler and @GuenterQuast are fine with this as well.

Reply:
Thanks for the notice, I've added them to the file